src/masker.js

```diff
@@ -77,16 +77,17 @@ export function formatter(value, config) {
     const masker = tokens[maskChar]
     let char = value[valueIndex]
 
+    // when is escape char, do not mask, just continue
+    if (masker && masker.escape) {
+      escaped = true
+      maskIndex++
+      continue
+    }
+
     // no more input characters and next character is a masked one
-    if (!char && masker) break
+    if (!char && masker && !escaped) break
```

Comment:
Move this if statement inside the one below, but put it after the `if (masker.escape) {` statement; and obviously it could now be changed to `if (!char) break` since the other logic is already been done in the parent if statement.